packages/office-ui-fabric-react/src/components/Pivot/examples/Pivot.AlwaysRender.Example.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+
+const ExpensiveToMount: React.FC = () => {
+  const [mounted, setMounted] = React.useState(false);
+  React.useEffect(() => {
+    setTimeout(() => {
+      setMounted(true);
+    }, 3000);
+  }, []);
+  return <div>{mounted ? 'Rendered' : 'Mounting...'}</div>;
+};
+
+export const PivotRenderActiveOnlyExample: React.FC = () => {
+  const [alwaysRender, setAlwaysRender] = React.useState(false);
+
+  const toggleActiveOnly = () => {
+    setAlwaysRender(!alwaysRender);
+  };
```

Comment:
You can use the `useBoolean` hook from `@uifabric/react-hooks` to manage this for you.